src/env.ts

```diff
@@ -0,0 +1 @@
+export const MONGO_URL = 'mongodb+srv://Ozokino:BLcRHxFAZylQHqKZ@taskdata.mzba1uw.mongodb.net/?retryWrites=true&w=majority&appName=TaskData';
```

Comment:
add this file to gitignore as I said before this contains sensitive data, so it shouldn't show anywhere in git history